vendor/github.com/digitalocean/godo/partner_interconnect_attachments.go

```diff
@@ -60,7 +61,7 @@ type partnerInterconnectAttachmentRequestBody struct {
 	// VPCIDs is the IDs of the VPCs to which the Partner Interconnect Attachment is connected
 	VPCIDs []string `json:"vpc_ids,omitempty"`
 	// BGP is the BGP configuration of the Partner Interconnect Attachment
-	BGP *BGP `json:"bgp,omitempty"`
+	BGP *BGPInput `json:"bgp,omitempty"`
```

Comment:
this is not what's in the latest go version 🤔 it was renamed to `partner_network_connect` in v1.140

Reply:
oh wait, we haven't release that change yet.

Reply:
let me cut a new release of godo real quick